content/src/main/content/jcr_root/apps/acs-commons/components/utilities/packager/asset-packager/configuration/dialog.xml

```diff
@@ -0,0 +1,140 @@
+                            value="MergePreserve"/>
+                    </options>
+                </package-acl-handling>
+                <conflict-resolution
+                    jcr:primaryType="cq:Widget"
+                    defaultValue="IncrementVersion"
+                    fieldDescription="Determines how package creation will be handled in the event of an existing package of the same Package group, name and version. [ Default: IncrementVersion ]"
+                    fieldLabel="Conflict resolution"
+                    name="./conflictResolution"
+                    type="select"
+                    xtype="selection">
+                    <options jcr:primaryType="cq:WidgetCollection">
+                        <replace
+                            jcr:primaryType="nt:unstructured"
+                            text="Replace"
+                            value="Replace"/>
+                        <increment-version
+                            jcr:primaryType="nt:unstructured"
+                            text="Increment Version"
+                            value="IncrementVersion"/>
+                        <none
+                            jcr:primaryType="nt:unstructured"
+                            text="None"
+                            value="None"/>
+                    </options>
+                </conflict-resolution>
+                <page-path
+                    jcr:primaryType="cq:Widget"
+                    fieldDescription="The parent page path to search for assets."
+                    fieldLabel="Page Path"
+                    name="./pagePath"
+                    rootPath="/content"
+                    xtype="pathfield"/>
+                <exclude-pages
+                    jcr:primaryType="cq:Widget"
+                    defaultValue="false"
+                    fieldDescription="Exclude the pages from the package definition. Creates a packages with just assets. [ Default: false ]"
+                    fieldLabel="Exclude Pages"
+                    inputValue="true"
+                    name="./excludePages"
+                    type="checkbox"
+                    xtype="selection"/>
+            </items>
+        </tab1>
+    <tab2
+        jcr:primaryType="cq:Widget"
+        title="Advanced"
+        xtype="panel">
+        <items jcr:primaryType="cq:WidgetCollection">
+            <asset-prefix
+                jcr:primaryType="cq:Widget"
+                fieldDescription="The path that the asset reference has to start with."
+                fieldLabel="Asset Prefix"
+                name="./assetPrefix"
+                rootPath="/content/dam"
+                xtype="pathfield"/>
+            <page-exclusions
+                jcr:primaryType="cq:Widget"
+                fieldDescription="Click the '+' to add a new exclude pattern. The pattern will both exclude the pages from the package and exclude them from the asset search.&lt;br/>By default all pages are searched and included in the package."
```

Comment:
Might be more clear to say "... pattern will exclude both the pages and their assets from the package.<br/>By default..."

Reply:
Addressed!